### ISSUE TITLE
Accordions wide gap

### ISSUE THREAD

**Issue by adiridho13** [tinymce/tinymce#9630]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud) or similar.
1. Insert accordion
2. Inside accordion, insert table
3. in each row table, add accordion
4. Click "togle accordion" to un-expand the section (refer to image accordion-uncollapse.png)
![accordion-uncollapse](https://github.com/tinymce/tinymce/assets/23132770/927ba844-a1d1-40d9-a3ce-061c986d49f4)

6. expand all section (refer to image collapsible expanded.png), 
![collapsible expanded](https://github.com/tinymce/tinymce/assets/23132770/8028a99a-83bb-45f8-9397-3b132d2074d2)

8. Try to drag row table  between collapsible 1 and 2 to top little bit (refer to image row-table-each-section.png)
![row-table-each-section](https://github.com/tinymce/tinymce/assets/23132770/785adfe8-f2a7-4450-8ec6-361842d73bb0)

10.  un-expand all section Collapsible 1 until collapsible 4

### ✔️ Expected result
Between all collapsible should not have wide gaps (refer to image: wide-gaps.png)
![wide-gaps](https://github.com/tinymce/tinymce/assets/23132770/2a5c4e34-041c-43cd-a0e9-ac4a5f552c64)
before row dragged, tr tag doesn't have css height:
![image](https://github.com/tinymce/tinymce/assets/23132770/fde52c42-f634-449a-90a8-3e52fafbac48)

### ❌ Actual result
there's auto adjustment height, so when accordion un-collapsed, there is a wide gaps between all section

After row dragged:
in source code I see auto adjusted css height: 
![image](https://github.com/tinymce/tinymce/assets/23132770/1c51911a-a615-4303-8da6-f3dfb9e4d0fb)

![image](https://github.com/tinymce/tinymce/assets/23132770/ea0bcbbb-c415-466a-8f9e-1ba0e3b38863)

## 📃 Other details
- Browser: Chrome
- OS Windows
- First affected version: latest
- Worked in version: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

Templated used:
<div class="content bullet-container jsCSS" align="left">
<table style="border-collapse: collapse; width: 1100px;" border="1">
<tbody>
<tr>
<td style="width: 1099px; background-color: #ffffff;">
<div class="starhub">
<details open="open">
<summary class="sh-summary"><strong><span class="sh-summary-title">Collapsible 1</span></strong></summary>
<div class="sh-summary-content"><br>
<table style="border-collapse: collapse; width: 1000.0px;" border="1">
<tbody>
<tr>
<td style="width: 998px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
</tbody>
</table>
</div>
</details>
</div>
</td>
</tr>
<tr>
<td style="width: 1099px; background-color: #ffffff;">
<div class="starhub">
<details open="open">
<summary class="sh-summary"><strong><span class="sh-summary-title">Collapsible 2</span></strong></summary>
<div class="sh-summary-content"><br>
<table style="border-collapse: collapse; width: 1000.0px;" border="1">
<tbody>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
</tbody>
</table>
</div>
</details>
</div>
</td>
</tr>
<tr>
<td style="width: 1099px; background-color: #ffffff;">
<div class="starhub">
<details open="open">
<summary class="sh-summary"><strong><span class="sh-summary-title">Collapsible 3</span></strong></summary>
<div class="sh-summary-content"><br>
<table style="border-collapse: collapse; width: 1000.0px;" border="1">
<tbody>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
<tr>
<td style="width: 998.0px; background-color: #ffffff;">
<div class="sh-summary-content">
<details open="open">
<summary class="sh-summary">Insert title here</summary>
<div class="sh-summary-content" style="padding-left: 40px;">Insert content here</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
<div class="sh-summary-content" style="padding-left: 40px;">&nbsp;</div>
</details>
<p>This part is outside of the collapsible area. Start here or delete if needed.</p>
</div>
</td>
</tr>
</tbody>
</table>
</div>
</details>
</div>
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
</div>
</details>
</div>
</div>
</div>
</div>
<div>
<div class="question question-click">
<div class="row">
<div class="acc-icon">&nbsp;</div>
<div class="col-md-20 col-xs-19 ques">

when I copy paste all source code to here, it showing normal without wide gaps

Tested in latest version: 
https://www.tiny.cloud/docs/tinymce/latest/accordion/

![image](https://github.com/tinymce/tinymce/assets/23132770/440dc280-55f5-4d6c-93db-c70fdca6d910)

**Comment by lostkeys** [comment]
Hi @adiridho13. If you touch the rows it will resize and set a height to the row. If you don't want that, use `table_resize_bars: false`.

**Comment by adiridho13** [comment]
> Hi @adiridho13. If you touch the rows it will resize and set a height to the row. If you don't want that, use `table_resize_bars: false`.

is it possible to make table auto fit the content ?

**Comment by lostkeys** [comment]
No, there is no button that does that. But you could build one using the APIs

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.
